.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.vscode
```

Comment:
If you change settings in the Workspace tab, such that they only apply for this repo, it will generate a `.vscode` folder. We don't want this checked in